src/locale/vi_VN.ts

```diff
@@ -0,0 +1,65 @@
+import { SuppportedContentLangId } from "./supported-content-lang-id";
+
+export const vi_VN = {
```

Comment:
Make sure it extends `typeof zh_CN` (like what is done in `en.ts`) and pass the typecheck

Reply:
_Pass typecheck_ = `pnpm exec tsc` gives no error